.github/workflows/release.yml

```
@@ -36,44 +38,47 @@ jobs:
       - name: 📤 Create Sentry release
         env:
           SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          SENTRY_URL: ${{ env.SENTRY_URL }}
         run: |
-          # Extract version from tag
-          VERSION=${GITHUB_REF#refs/tags/v}
+          set -euo pipefail  # Bash safety flags
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)